ENIGMAsystem/SHELL/Platforms/Win32/WINDOWSexternals.cpp

```diff
@@ -115,20 +114,20 @@ int external_define(string dll,string func,int calltype,bool returntype,int argc
   	dllmod = LoadLibrary(dll.c_str());
   else
   {
-    printf("LOADING PREEXISTING HANDLE");
+    DEBUG_MESSAGE("LOADING PREEXISTING HANDLE", MESSAGE_TYPE::M_WARNING);
     dllmod = dllHandles[dll];
   }
 
   if (dllmod == NULL)
   {
-    show_error(std::string("Cannot load library \"") + dll + std::string("\"!"), 0);
+    DEBUG_MESSAGE("Cannot load library " + dll, MESSAGE_TYPE::M_ERROR);
```

Comment:
Why delete the quotes? It really sucks when the error just says `Cannot load library ` instead of `Cannot load library ""`.